evap/staff/tests/test_views.py

```diff
@@ -322,16 +410,18 @@ def test_import_invalid_file(self):
 
         self.assertEqual(UserProfile.objects.count(), original_user_count)
 
-    def test_test_run(self):
+    def test_import_wrong_file(self):
         page = self.app.get(self.url, user='staff')
 
-        original_participant_count = self.course.participants.count()
+        original_user_count = UserProfile.objects.count()
 
         form = page.forms["semester-import-form"]
-        form["excel_file"] = (self.filename_valid,)
-        form.submit(name="operation", value="test")
+        form["excel_file"] = (self.filename_random,)
+        reply = form.submit(name="operation", value="test")
 
-        self.assertEqual(self.course.participants.count(), original_participant_count)
+        self.assertEqual(UserProfile.objects.count(), original_user_count)
+        self.assertContains(reply, "Couldn&#39;t read the file. Error: Unsupported format, or corrupt file: Expected BOF record;")
+        self.assertNotContains(reply, 'Import previously uploaded file')
```

Comment:
OTOH, having one test that checks whether errors from the importer are properly handled by the view are good to have, too.